source/guide_flarum.rst

```diff
@@ -0,0 +1,75 @@
+.. highlight:: console
+
+.. author:: Thomas Kammerer <https://kammerer.tk/>
+
+.. tag:: lang-php
+.. tag:: web
+.. tag:: forum
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/flarum.png
+      :align: center
+
+######
+Flarum
+######
+
+.. tag_list::
+
+Flarum_ is the next-generation forum software that makes online discussion fun. It's simple, fast, and free.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`PHP <lang-php>`
+  * :manual:`MySQL <database-mysql>`
+  * :manual:`domains <web-domains>`
+
+Prerequisites
+=============
+
+We’re using :manual:`PHP <lang-php>` in the stable version 7.2. Since new Uberspaces are currently setup with PHP 7.1 by default you need to set this version manually:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version use php 7.2
+ Selected PHP version 7.2
+ The new configuration is adapted immediately. Patch updates will be applied automatically.
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+Your website domain needs to be set up:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+We create the database and install Flarum using composer.
+
+ [isabell@stardust ~]$ mysql -e "CREATE DATABASE ${USER}_flarum"
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html
+ [isabell@stardust html]$ composer create-project flarum/flarum . --stability=beta
```

Comment:
The [docs suggest](https://flarum.org/docs/install.html#url-rewriting) that only `/public` should be served via the webserver, but this serves everything. This also exposes the content of `storage/logs` to the internet, which is probably not good. You can try something like this to change that:

```
cd /var/www/virtual/$USER/
composer create-project flarum/flarum flarum --stability=beta
rmdir html
ln -s flarum html
```